packages/backend/Dockerfile

```diff
@@ -4,10 +4,12 @@ WORKDIR /workspace
 
 RUN apt update
 RUN apt install -y curl xz-utils
+COPY package.json yarn.lock tsconfig.build.json .yarnrc.yml ./
```

Comment:
These changes remove reliance on the [yarn-docker-build](https://github.com/Dcard/yarn-plugins/tree/master/packages/docker-build/src) plugin, which made it impossible to build with buildx for different architectures.